packages/xod-client/src/user/actions.js

```diff
@@ -1,12 +1,12 @@
 import url from 'url';
 import { UPDATE_COMPILE_LIMIT } from './actionTypes';
-import { ORIGIN_COMPILE } from './constants';
+import { XOD_COMPILE_DOMAIN } from './constants';
 
 export const updateCompileLimit = () => (dispatch) => {
   fetch(url.format({
     ...url.parse(process
-      ? process.env.ORIGIN_COMPILE || ORIGIN_COMPILE
-      : ORIGIN_COMPILE),
+      ? process.env.XOD_COMPILE_DOMAIN || XOD_COMPILE_DOMAIN
+      : XOD_COMPILE_DOMAIN),
```

Comment:
There should not be a dedicated constant for compilation URL. Please, use a single env variable with fallback to “xod.io” if not set.

There is already `utils/siteLinks.js` where you can do something like:

```js

const HOSTNAME = process.env.XOD_HOSTNAME || 'xod.io';

export const compileServiceHostname = `https://compile.${HOSTNAME}`;
export const authServiceHostname = `https://auth.${HOSTNAME}`;
// etc
```

In that case, all entry point URL stuff would be in a single place and easily customizable. `siteLinks` is not the best name if we manage service URL’s too, but it could be easily renamed to something like `urls.js`